### PR TITLE
Feature/comp shuffleboard

### PR DESCRIPTION
**Pull request by amadeawagner**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 1]
```diff
@@ -109,8 +109,18 @@ public class SwerveSubsystem extends SubsystemBase{
...
         controlOrientationIsFOD = true;
+        
+        if (!Robot.isCompetition){
+            matchTab = Shuffleboard.getTab("Match");
+            controlOrientationEntry = matchTab.add("FOD", true).getEntry();
+            headingEntry = matchTab.add("NavX Yaw", 0).withWidget(BuiltInWidgets.kGyro).getEntry();
+            pitchEntry = matchTab.add("NavX Pitch", 0).withWidget(BuiltInWidgets.kGyro).getEntry();
+            configTab = Shuffleboard.getTab("Config");
+            positionEntry = configTab.add("Position", "").getEntry();
+        };
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/subsystems/SwerveSubsystem.java:123]: Check in periodic and also disable the updates of these entries.

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 2]
```diff
@@ -109,8 +109,18 @@ public class SwerveSubsystem extends SubsystemBase{
...
 
         //Add coast mode command to shuffleboard
         configTab.add(new StartEndCommand(this::coastModules, this::brakeModules, this).ignoringDisable(true).withName("Coast Modules"));
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/subsystems/SwerveSubsystem.java:126]: DO we need this in competition?  CHeck with Drive team ( Michael and Zane )

File: src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java [hunk 3]
```diff
@@ -50,10 +51,15 @@ public class BuddyBalanceSubsystem extends SubsystemBase {
...
-  private ShuffleboardTab buddyBalanceTab = Shuffleboard.getTab("Buddy Balance"); // TODO: Replace buddy balance tab with whatever tab the position should be logged to
+  private ShuffleboardTab buddyBalanceTab; // TODO: Replace buddy balance tab with whatever tab the position should be logged to
   private GenericEntry positionEntry;
 
   public BuddyBalanceSubsystem() {
+
+    if (!Robot.isCompetition){
+      buddyBalanceTab = Shuffleboard.getTab("Buddy Balance");
+    }
+
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java:62]: positionEntry = line below this line....

Do we need that in competition?

**Review by caseyjbrotherton — CHANGES_REQUESTED** [review]
Need to check each periodic function and where all of the shuffleboard objects are referenced after initializing.  

ALso need to check and see what needs to be kept for competition in each of the subsystems.

ALso, please check OI.java as well

**Review by zbrotherton — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 4]
```diff
@@ -98,7 +99,10 @@ public class SwerveSubsystem extends SubsystemBase{
     private GenericEntry pitchEntry = matchTab.add("NavX Pitch", 0).withWidget(BuiltInWidgets.kGyro).getEntry();
 
     private ShuffleboardTab configTab = Shuffleboard.getTab("Config");
-    private GenericEntry positionEntry = configTab.add("Position", "").getEntry();
+    //private GenericEntry positionEntry = configTab.add("Position", "").getEntry();
+    
+
```
Comment (azvanderpas) [inline src/main/java/frc/robot/subsystems/SwerveSubsystem.java:104]: Remove commented out lines

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 5]
```diff
@@ -278,10 +284,14 @@ public Command followTrajectory(String name, PathPlannerTrajectory trajectory){
...
-        pitchEntry.setDouble(gyro.getPitch());
-        headingEntry.setDouble(getHeading());
+
         positionEntry.setString(getPose().getTranslation().toString());
+        
+        if (!Robot.isCompetition) {
+            pitchEntry.setDouble(gyro.getPitch());
+            headingEntry.setDouble(getHeading());
+        }
+
```
Comment (azvanderpas) [inline src/main/java/frc/robot/subsystems/SwerveSubsystem.java:294]: I think we'd want to keep at leas the headingEntry logging

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 6]
```diff
@@ -36,7 +38,6 @@ public IntakeSubsystem() {
   private void spinWheels(double velocity) {
     motor.set(velocity);
     intakeEntry.setDouble(velocity);
-
```
Comment (azvanderpas) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java:39]: this file can be reverted, since there are only whitespace changes

**Review by azvanderpas — CHANGES_REQUESTED** [review]

**Pull request closed without merging**